src/index.html

```diff
@@ -1,12 +1,286 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="nav">
+      <img class="nav__logo" src="./images/logo.png"
+      alt="logo">
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__list-part">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__list-part">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__list-part">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__list-part">
+            <a href="#"
+            class="nav__link"
+            data-qa="hover">
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__list-part">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__list-part">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__list-part">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__list-part">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="grid">
```

Comment:
This BEM is not proper, should be `cards` with elements `cards__card` for example. Now elements don't belong to any block.